### DISCUSSION TITLE
Bosnian language support

### DISCUSSION THREAD

**Discussion by emingratz** [directus/directus#15893] [Feature Requests]
Hello guys.
I'm glad I have discovered this wonderful piece of software. I gave it a try, and noticed that both Croatian and Serbian language are supported. I would like to see Bosnian as well. It's quite similar to these mentioned above.
Long story short, my proposition is this: you clone Croatian and rename it to Bosnian (bs_BA).
Then you send me a document with all Croatian words/strings, and I will replace it with Bosnian words accordingly, where needed. Then I will send the document back to you. I would like to work on the document level, and not to push or pull repositories, etc.
I am ready to contribute, so please let me know if I can be included, and what are the further steps.
Thanks,
Emin

**Comment by rijkvanzanten** [comment]
Hi Emin!

I've enabled Bosnian on our Locales platform https://locales.directus.io . You can start adding the translations there :) I'll add a PR here to enable Bosnian to be selected in app once translations exist on locales.directus.io 👍🏻

**Reply by emingratz** [reply]
Hey @rijkvanzanten, thank you for your response. It means a lot!
I will start work on it ASAP.
Regards!

**Reply by emingratz** [reply]
It is done, my friend. The Bosnian translation (98%) now lives on https://locales.directus.io/.
Cheers. Let me know when the cake is baked. :-)

**Reply by benhaynes** [reply]
Oh my, thank you!!! ❤️

**Reply by rijkvanzanten** [reply]
Cake is baked and will be released soon 🙂 https://github.com/directus/directus/pull/15993

**Reply by emingratz** [reply]
> Cake is baked and will be released soon slightly_smiling_face #15993

Yes! Thanks a lot, man. I am looking forward to it.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
